packages/ladle/lib/cli/vite-plugin/parse/get-entry-data.js

```diff
@@ -28,6 +28,31 @@ export const getEntryData = async (entries) => {
   return entryData;
 };
 
+export const getNamedExportVariables = (
+  /** @type {string[] | undefined} */ lines,
+) => {
+  /**
+   * @type {string[]}
+   */
+  let namedExports = [];
+  lines?.forEach((line) => {
+    let namedExport = "";
+    // https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export
+    if (line?.startsWith("export", 0) && !line?.includes("default")) {
```

Comment:
What if there are multiple stories in one file? What if users is exporting some constant in between the stories? Anyway, this should not be needed since we are already getting this information [here](https://github.com/tajo/ladle/blob/main/packages/ladle/lib/cli/vite-plugin/parse/get-named-exports.js#L76) using AST traversing.